### PR TITLE
quick fix for dc->deepchem

### PR DESCRIPTION
**Pull request by miaecle**
Sorry for the miss, did not notice the dc problem, is it fine to `import deepchem` at the beginning and use function as `deepchem.XX.XX` instead of `from deepchem.XX import XX` at the top?
#451

### CODE REVIEW

**Comment by rbharath** [comment]
@miaecle Thanks for the quick fix. Would it be feasible to add some simple unit tests for the `dc.molnet` module? Something that tests that at least the small datasets can be downloaded and loaded?

**Comment by XericZephyr** [comment]
Hi, @rbharath, that's a nice idea! It could be something I am expecting too.

But I can expect some huge engineering efforts behind it. Maybe we can start an issue and resolve it in some separate PRs.

**Comment by rbharath** [comment]
@XericZephyr I think this PR is a good place to add some unit tests.

**Comment by XericZephyr** [comment]
@rbharath  I was only hoping the `n_features` bug can be fixed real quick. I could personally check out the micheal's fork at this moment.

File: deepchem/molnet/run_benchmark.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def run_benchmark(datasets,
                   split=None,
```
Comment (XericZephyr) [inline deepchem/molnet/run_benchmark.py]: Can we accept list input of split, e.g., ["index", "random"]?

**Comment by XericZephyr** [comment]
On line 188 branch of `run_benchmark.py`. After `model.fit`, `train_scores` have not been initialized. I suppose we should definitely assign it as `dict` before.

**Comment by miaecle** [comment]
@XericZephyr Ah, I see. There is a bug here between namings, fixed.

File: deepchem/molnet/run_benchmark.py (hunk 1 above)
Comment (miaecle) [inline deepchem/molnet/run_benchmark.py]: Suggest you to do the loop outside the function.
Here we only accept multiple datasets for convenience, if multiple splits, models are all permitted, things could go too complex.

**Comment by rbharath** [comment]
@miaecle Thanks for the rapid-turnaround on this PR! Much appreciated.

@XericZephyr Thanks for the feedback and bug catches!

**Pull request merged**